aep/general/0003/aep.md

```diff
@@ -68,8 +68,7 @@ An object upon which one or more methods can operate.
 
 ### API Resource Type
 
-The type of an object exposed via an API. It is globally unique within an
-API.
+The type of an object exposed via an API. It is globally unique within an API.
```

Comment:
Is "object" the right word here?  An API resource isn't exactly the same thing as an "object" in OOP terms, even though there are some parallels, so this might be confusing in that sense.  Maybe "entity"?

Reply:
I just realized that we defined "API resource" above, so I'll just re-use that.

FWIW I used the term object to match Kubernetes: https://kubernetes.io/docs/concepts/overview/working-with-objects/. Terraform calls these resources, so I think we're already in a world where there's duplicate terminology.

But entity is as good as any - I updated the API resource above to use that term, and updated this section to use the term "API resource".